test/unit/dissemination-test.js

```diff
@@ -70,3 +70,80 @@ testRingpop('avoids redundant dissemination by filtering changes from source', f
         membership.checksum);
     assert.ok(changes.length > 0, 'changes issued');
 });
+
+testRingpop('raise piggyback counter on issueAsReceiver', function t(deps, assert) {
+    var ringpop = deps.ringpop;
+    var membership = deps.membership;
+    var dissemination = deps.dissemination;
+
+    var addrAlive = '127.0.0.1:3001';
+    var addrSuspect = '127.0.0.1:3002';
+    var addrFaulty = '127.0.0.1:3003';
+    var incNo = Date.now();
+
+    // Clear changes to start fresh, otherwise local member changes
+    // recorded during bootstrap phase would have been issued.
+    dissemination.clearChanges();
+
+    membership.makeAlive(addrAlive, incNo);
+    membership.makeSuspect(addrSuspect, incNo);
+    membership.makeFaulty(addrFaulty, incNo);
+
+    // 'sender' and source of updates are different; issues changes.
+    var changes;
+
+    assert.equal(dissemination.maxPiggybackCount, 15, 'assert maxPiggybackCount is 15');
```

Comment:
I would suggest not referring to a magic number like `15` here. It leads to brittle assertions. I'd get rid of this assertion altogether or extract a method from `adjustMaxPiggybackCount` to compute the piggyback count based on known cluster size.